app/models/hostgroup.rb

```diff
@@ -113,15 +113,16 @@ def self.title_name
     "title".freeze
   end
 
-  def disk_layout_template
-    if ptable.present?
-      { name: ptable.name, template: ptable.layout }
-    end
+  def disk_layout_source
+    @disk_layout_source ||= if ptable.present?
+                              Foreman::Renderer::Source::String.new(name: ptable.name,
+                                                                    content: ptable.layout.tr("\r", ''))
+                            end
   end
 
   def diskLayout
-    raise Foreman::Renderer::RenderingError, 'Partition table not defined for hostgroup' unless disk_layout_template
-    disk_layout_template[:template].tr("\r", '')
+    raise Foreman::Renderer::Errors::RenderingError, 'Partition table not defined for hostgroup' unless disk_layout_source
```

Comment:
this exception seems to hardcode MESSAGE constant and can't be customized from point where it's raised

Reply:
oops sorry, ignore me, I misread the code